flux-init/init-dev/gotk-repo.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: main
+    branch: feature/flux-dashboards
```

Comment:
I dont think we want to change the branch as part of PR

Reply:
I was running the dashboards in one of our clusters @joaquinrz it is back to main now